src/templates/Header/index.js

```diff
@@ -1,13 +1,12 @@
-/* eslint react/prop-types: 0 */
-
 import React, { useEffect } from 'react'
 import { connect } from 'react-redux'
+import { bool, func, string } from 'prop-types'
 
 import Preloader from 'ui-kit/preloader/preloader'
```

Comment:
I'd expect to have this kind of import:
```
import { Preloader } from 'ui-kit'
```
can you please make this change now and in the future as well ? 
whenever you add a component in **ui-kit**, imports should always look cleaner as the one from above